packages/shared/components/icon/icons.js

```diff
@@ -1174,4 +1174,15 @@ export const icons = {
             },
         ],
     },
+    search: {
+        width: 20,
+        height: 20,
+        path: [
+            {
+                d: 'M11.1922 12.6064C10.0236 13.4816 8.57236 14 7 14C3.13401 14 0 10.866 0 7C0 3.13401 3.13401 0 7 0C10.866 0 14 3.13401 14 7C14 8.57232 13.4816 10.0236 12.6064 11.1921L16.0001 14.5858C16.3906 14.9763 16.3906 15.6095 16.0001 16C15.6096 16.3905 14.9764 16.3905 14.5859 16L11.1922 12.6064ZM12 7C12 9.76142 9.76142 12 7 12C4.23858 12 2 9.76142 2 7C2 4.23858 4.23858 2 7 2C9.76142 2 12 4.23858 12 7Z',
```

Comment:
Im going to update the path of this icon to follow the rules of the other icons in the UI Kit